src/prefect/cli/server.py

```diff
@@ -225,22 +226,21 @@ async def start(
         except Exception:
             pass
 
-    server_env = os.environ.copy()
-    server_env["PREFECT_API_SERVICES_SCHEDULER_ENABLED"] = str(scheduler)
-    server_env["PREFECT_SERVER_ANALYTICS_ENABLED"] = str(analytics)
-    server_env["PREFECT_API_SERVICES_LATE_RUNS_ENABLED"] = str(late_runs)
-    server_env["PREFECT_API_SERVICES_UI"] = str(ui)
```

Comment:
While doing this, I discovered that `PREFECT_API_SERVICES_UI` isn't a thing, so I removed it.  Not sure if was supposed to be another setting or just an old variable?

Reply:
Yeah, `PREFECT_UI_ENABLED` is the correct one, which also there, so this is good to remove.